libs/entry-components/search-filter/entry-search-filter.component.ts

```diff
@@ -69,5 +72,13 @@ export class EntrySearchFilterComponent implements OnInit {
   asDateSearchFilter<T>(searchFilter: SearchFilterBase<T>): DateSearchFilter<T> {
     return searchFilter as DateSearchFilter<T>;
   }
-}
 
+  setServerErrors(errors: { [key: string]: string[] }) {

```

Comment:
There is a helper method that does this (this code can be removed) See: https://github.com/enigmatry/entry-angular-building-blocks/blob/9df5d90230a79adca8f5d8d79771b4f12dd44cdc/libs/entry-components/validation/entry-validation.ts#L29

(method that maps received server side validation errors to the form and its fields.)